DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  CalculatorViewController.swift
+//  DhawaAlkhreej
+//
+//  Created by FarouK on 21/05/2019.
+//  Copyright © 2019 DhawaAlkhreej. All rights reserved.
+//
+
+import UIKit
+
+class CalculatorVC: BaseViewController {
+
+    @IBOutlet var ThanwiyahLabel: UILabel!
+    @IBOutlet var quodratLabel: UILabel!
+    @IBOutlet var tahsilyLabel: UILabel!
+    @IBOutlet var stepExamLabel: UILabel!
+    @IBOutlet var calculateButton: UIButton!
+    @IBOutlet var resultLabel: UILabel!
+    
+    @IBOutlet var requestedThanawiyahTF: DesignableTF!
+    
+    var percantage = ["30%", "40%", "50%"]
+    var selectedPercentage: String?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        createPercentagePicker()
+        createPickerToolbar()
+        ThanwiyahLabel.font = Identity.font(.custom(weight: .bold, size: 20))
```

Comment:
Please extract  the ui related code to its own method

Reply:
I will, it was just in initial implementation to test out things.